zyte_common_items/serialization.py

```diff
@@ -0,0 +1,105 @@
+"""Web-poet serialization support for Zyte Common Items.
+
+This module registers serialization and deserialization functions for
+top-level Item classes with web-poet's serialization system, enabling support
+for scrapy savefixture and other serialization use cases.
+
+Registers serialization for all top-level items (one per file) in the
+zyte_common_items.items module, excluding custom_attributes.py.
+"""
+
+import json
+from typing import Type
+
+import itemadapter
+from web_poet.serialization import SerializedLeafData, register_serialization
+
+from zyte_common_items.adapter import ZyteItemAdapter
+from zyte_common_items.base import Item
+
+
+class ZCEItemAdapter(itemadapter.ItemAdapter):
+    """ItemAdapter configured to use ZyteItemAdapter for serialization."""
+
+    ADAPTER_CLASSES = [ZyteItemAdapter]
+
+
+def _create_serialization_functions(item_class: Type[Item]):
```

Comment:
I wonder if we can just have a single function for Item, which is the base class. Probably it'd work (I think singledispatch supports it), without a need to register serialization/deserialization for individual classes.